web/src/app/page.tsx

```diff
@@ -1,47 +1,34 @@
-// const cmsContentType = 'announcements';
-// fetch(`${process.env.CMS_API}/${cmsContentType}`, {
-//     headers: {
-//         authorization: `Bearer ${process.env.CMS_TOKEN}`,
-//     },
-// })
-//     .then((ok) => ok.json())
-//     .then((res) => console.log(res));
+"use client";
+import { useEffect, useState } from "react";
+import Hero from "@/components/Strapi/Sections/HeroSection";
+import PageContent from "@/components/Strapi/Sections/Content";
+import { PageAttributes } from "@/types/strapi";
 
 export default function Home() {
+	const [pageData, setPageData] = useState<PageAttributes | undefined>(undefined);
+
+	useEffect(() => {
+		const fetchData = async () => {
```

Comment:
We could also think about using [SWR](https://www.npmjs.com/package/swr) for data fetching

Reply:
I had to do this because we had some pipeline issues regarding our build step. The API was not available there and I had to implement a route handler to fetch it without issues.

Reply:
Yes SWR is also for client components but offers a very nice UX. Maybe take a look. It's also recommended by Next.js

Reply:
Implemented it in a very simple way